include/native/parallel_execution_native.h

```diff
@@ -1464,6 +1446,11 @@ void parallel_execution_native::do_pipeline_nested(
       std::forward<Transformers>(std::get<I>(transform_ops))...);
 }
 
+template<typename T, typename... Others>
+void parallel_execution_native::do_pipeline(mpmc_queue<T> &, mpmc_queue<T> &, Others &&...) const
```

Comment:
Where does this one come from?
Wasn't it in the class delcaration?

Reply:
It was in the class declaration but due to the parameters not being used, and for documentation purpose we had to move it outside so that the parameters inside the class could have names and not raise warning by the compilers